test/external/external_test_onnx_backend.py

```diff
@@ -90,7 +90,6 @@ def supports_device(cls, device: str) -> bool:
 
 # no scatternd gathernd
 backend_test.exclude('test_gathernd_*')
```

Comment:
there are scatter tests in onnx but voodoo magic is making those tests disappear
I checked that scatter tests are indeed in `test_cases` for 
`globals().update(backend_test.enable_report().test_cases)` (this is after all the excludes)
but somehow it's just not being tested...

gonna investigate this further.